src/AdTypes.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\fb_instant_articles_display\DrupalInstantArticleDisplay.
+ */
+
+namespace Drupal\fb_instant_articles;
+
+abstract class AdTypes {
```

Comment:
@simonengelhardt is this so you can get constants both within the procedural form functions and from within the `addAdsFromSettings()` method? If so I can see this. Are these ad types likely to change?